### ISSUE TITLE
Value is read out

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#107663]
Testing #107610

* ubuntu20 w/ orca
* navigate to "Editor: Format on Save"
* :bug: it doesn't read the "value" if the setting is check or not. It reads the whole description and only when pressing enter it reads "checked" or "unchecked"

**Comment by roblourens** [comment]
I don't know whether it should or not. Not all values can easily be read in that context (objects, arrays), some may be long, or confusing outside of the context of focus inside the control. I think it makes sense to interact with the control to get the value. @isidorn ?

**Comment by isidorn** [comment]
For a boolean setting what Joh is suggesting makes sense, but for consistency if we do it for boolean settings we should do for all.
I suggest that for now leave this as it is. And if we also get feedback from the community that mathces this we can append the value to the top aria label.

**Comment by roblourens** [comment]
I agree with that. Will listen for more feedback.
